Userland/Utilities/grep.cpp

```diff
@@ -195,25 +197,28 @@ ErrorOr<int> serenity_main(Main::Arguments args)
 
         auto handle_file = [&matches, binary_mode, suppress_errors, count_lines, quiet_mode,
                                user_specified_multiple_files, &matched_line_count, &did_match_something](StringView filename, bool print_filename) -> bool {
```

Comment:
Making `handle_file` and `handle_directory` return an `ErrorOr<>` sounds low-impact enough that we could just do it right now instead of spewing `release_value_but_fixme_should_propagate_errors` everywhere.